python-backend/app/api/mines/incidents/resources/mine_incident_determination_types.py

```diff
@@ -0,0 +1,15 @@
+from flask_restplus import Resource
+
+from app.extensions import api
+from app.api.utils.access_decorators import requires_role_mine_view
+
+from ..models.mine_incident_determination_type import MineIncidentDeterminationType
+from ...mine_api_models import MINE_INCIDENT_DETERMINATION_TYPE_MODEL
+
+
+class MineIncidentDeterminationTypeResource(Resource):
+    @api.marshal_with(MINE_INCIDENT_DETERMINATION_TYPE_MODEL, envelope='options', code=200, as_list=True)
```

Comment:
I know we have a number of endpoints that return code table dropdown options as `options`, but I believe we've agreed to move away from that pattern. New/refactored endpoints are using `records`, which can be standard across all list endpoints

Reply:
Thanks, fixed 👍